README.md

```diff
@@ -1,2 +1,49 @@
 # Awesome_Books
 Plain JavaScript with Objects 
+![screenshot](https://user-images.githubusercontent.com/84907743/137748695-d285bb7e-ac14-4295-88a0-b1677098338d.png)
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+In vsCode console run $git clone 
+Navigate to the index.html page and open it with live server
```

Comment:
- [**_NOT REALLY OPTIONAL_**] Our README should be as descriptive as if to a person with no technical know-how. This **_`getting started`_** is not descriptive enough. Please do something about it in the upcoming milestones of this project. Tell the user in simpler terms:

- How to clone the repo
- How to navigate to the project's directory
- How to run the project/app

One last thing, the README can't be complete without the **_`about project`_** section. Please add this section and describe what this project/app is all about.